### PR TITLE
Improving Pearson default theme

### PR DESCRIPTION
**Pull request by alexjmpb**
This PR aims to make some improvements in the pearson default theme to make it consistent with the MFEs.

## Changes

- [x] Adjust header and footer styles and structure.
- [x] Make show programs link configurable from site configurations.

## Screenshots
Screenshot from 2023-06-30 16-46-55
Screenshot from 2023-06-30 16-46-50

## How to test

- Checkout to this branch
- Enable comprehensive theming
- Add pearson-theme to a site theme
- Check that the changes are applied correctly

## Reviewers

- [ ] @kuipumu 
- [ ] @sergivalero20 
- [ ] @anfbermudezme

### CODE REVIEW

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_courses.scss [hunk 1]
```diff
@@ -718,3 +718,13 @@ div.content-wrapper.main-container div.container div.home {
...
+
+.courses-container .courses .course .course-info {
+  display: none !important;
+}
+
+@media (max-width: 540px) {
+  .courses {
+    width: 100%;
+  }
+}
```
Comment (sergivalero20) [inline edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_courses.scss]: @alexjmpb add a new line here, please.

File: edx-platform/pearson-theme/lms/templates/footer.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  <%static:js group='base_vendor'/>
+  <%static:css group='style-vendor'/>
+  <%include file="widgets/segment-io.html" />
+  <%include file="widgets/segment-io-footer.html" />
+% endif
+% if footer_css_urls:
+  % for url in footer_css_urls:
+    <link rel="stylesheet" type="text/css" href="${url}"></link>
+  % endfor
+% endif
```
Comment (sergivalero20) [inline edx-platform/pearson-theme/lms/templates/footer.html]: @alexjmpb add a new line here, please.

File: edx-platform/pearson-theme/lms/templates/header/header.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                        <option value="${language[0]}" selected="selected">${language[1]}</option>
+                    % else:
+                        <option value="${language[0]}" >${language[1]}</option>
+                    % endif
+                % endfor
+            </select>
+            </label>
+        </form>
+    % endif
+% endif
```
Comment (sergivalero20) [inline edx-platform/pearson-theme/lms/templates/header/header.html]: @alexjmpb add a new line here, please.

File: edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_variables.scss [hunk 4]
```diff
@@ -1,6 +1,45 @@
...
+$serif: Playfair Display;
+
 // Initialize the Open edX Bootstrap theme
-@import 'edx-bootstrap/sass/open-edx/theme';
+
+//navigation
+$button-color-login: #003057;
+$font-color-login: #FFF;
+$button-color-register: #003057;
+$font-color-register: #FFF;
```
Comment (sergivalero20) [inline edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_variables.scss]: @alexjmpb add a new line here, please.

File: edx-platform/pearson-theme/lms/templates/header/header.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        % if user.is_authenticated:
+            <%include file="navbar-authenticated.html" args="online_help_token=online_help_token"/>
+        % else:
+            <%include file="navbar-not-authenticated.html" args="online_help_token=online_help_token"/>
+        % endif
+    </div>
+    <div class="mobile-menu hidden" aria-label=${_("More Options")} role="menu" id="mobile-menu"></div>
+</header>
+
+% if course:
```
Comment (sergivalero20) [inline edx-platform/pearson-theme/lms/templates/header/header.html]: @alexjmpb Is this conditional necessary?

**Review by anfbermudezme — APPROVED** [review]

**Review by sergivalero20 — APPROVED** [review]

**Pull request merged**